src/main/java/com/solvd/carina/demo/mobile/saucedemomobile/utils/AuthUtilsScreen.java

```diff
@@ -0,0 +1,19 @@
+package com.solvd.carina.demo.mobile.saucedemomobile.utils;
+
+import com.solvd.carina.demo.mobile.saucedemomobile.common.LoginScreenBase;
+import com.solvd.carina.demo.mobile.saucedemomobile.common.ProductsScreenBase;
+import com.zebrunner.carina.utils.R;
+import com.zebrunner.carina.utils.factory.ICustomTypePageFactory;
+
+public class AuthUtilsScreen implements ICustomTypePageFactory{
```

Comment:
I suppose it should not be screen, but just AuthUtil
```suggestion
public class AuthUtils implements ICustomTypePageFactory {
```

Reply:
If you have AuthUtils for web please consider renaming this class to MobileAuthUtils to avoid confusion